semconv/v1.27.0/generated_attribute_group.go

```diff
@@ -5,165 +5,217 @@
 
 package semconv
 
-// The Android platform on which the Android application is running.
+// Namespace: android
 const (
-	// Uniquely identifies the framework API revision offered by a version
-	// (os.version) of the android operating system. More information can be found
-	// here.
+
+	// Uniquely identifies the framework API revision offered by a version (`os.version`) of the android operating system. More information can be found [here].
 	//
+	// Stability: Experimental
 	// Type: string
-	// Requirement Level: Optional
-	// Stability: experimental
-	// Examples: '33', '32'
+	//
+	// Examples:
+	// "33",
+	// "32",
+	//
+	// [here]: https://developer.android.com/guide/topics/manifest/uses-sdk-element#ApiLevels
 	AttributeAndroidOSAPILevel = "android.os.api_level"
+
+	// Deprecated use the `device.app.lifecycle` event definition including `android.state` as a payload field instead.
+	//
+	// Stability: Experimental
+	// Type: Enum
+	// Deprecated: Replaced by `device.app.lifecycle`.
+	//
+	// Examples: undefined
+	// Note: The Android lifecycle states are defined in [Activity lifecycle callbacks], and from which the `OS identifiers` are derived
+	//
+	// [Activity lifecycle callbacks]: https://developer.android.com/guide/components/activities/activity-lifecycle#lc
+	AttributeAndroidState = "android.state"
+)
+
+// Enum values for android.state
+const (
+	// Any time before Activity.onResume() or, if the app has no Activity, Context.startService() has been called in the app for the first time
+	AttributeAndroidStateCreated = "created"
+	// Any time after Activity.onPause() or, if the app has no Activity, Context.stopService() has been called when the app was in the foreground state
+	AttributeAndroidStateBackground = "background"
+	// Any time after Activity.onResume() or, if the app has no Activity, Context.startService() has been called when the app was in either the created or background states
+	AttributeAndroidStateForeground = "foreground"
 )
 
-// This group describes attributes specific to artifacts. Artifacts are files
-// or other immutable objects that are intended for distribution. This
-// definition aligns directly with the
-// [SLSA](https://slsa.dev/spec/v1.0/terminology#package-model) package model.
+// Namespace: artifact
 const (
-	// The provenance filename of the built attestation which directly relates to the
-	// build artifact filename. This filename SHOULD accompany the artifact at publish
-	// time. See the SLSA Relationship specification for more information.
+
+	// The provenance filename of the built attestation which directly relates to the build artifact filename. This filename SHOULD accompany the artifact at publish time. See the [SLSA Relationship] specification for more information.
 	//
+	// Stability: Experimental
 	// Type: string
-	// Requirement Level: Optional
```

Comment:
requirement level disappeared

Reply:
Yes - There is not such things are requirement level on the attribute registry, that's signal specific.

I'm not even sure where it was pulling requirement level from before, but I call that out in the PR description.